test/session.js

```diff
@@ -1,51 +1,53 @@
+/* global describe, it, before */
```

Comment:
Rather than this, why not use the proposed pattern at https://github.com/expressjs/discussions/issues/18#issuecomment-223182041 ? If you don't like the proposed pattern, please speak up in that discussion thread :)!